Gutenberg/cocoapods_helpers.rb

```diff
@@ -172,3 +177,27 @@ def react_native_version!(gutenberg_path:)
 
   package_json_version.split('.').map(&:to_i)
 end
+
+# A workaround for the issue described at
+# https://github.com/wordpress-mobile/WordPress-iOS/pull/21504#issuecomment-1789466523
+#
+# For some yet-to-discover reason, something in the process installing the pods
+# using local sources messes up the LIBRARY_SEARCH_PATHS.
+def workaround_broken_search_paths
+  project = Xcodeproj::Project.open('WordPress/WordPress.xcodeproj')
+
+  library_search_paths_key = 'LIBRARY_SEARCH_PATHS'
+  broken_search_paths = '$(SDKROOT)/usr/lib/swift$(inherited)'
```

Comment:
Likewise, this hardcoded string makes the workaround work only in one particular case. But I don't think we need to make it more flexible. Better to find the root cause.